funnel/views/profile.py

```diff
@@ -22,6 +22,11 @@
 from .mixins import ProfileViewMixin
 
 
+@Profile.features('new_project')
+def feature_profile_new_project(obj):
+    return bool(obj.state.PUBLIC)
```

Comment:
Also test if the user has access. Remove that test from the template and do it here.